src/modules/minigame/fire_fighters/fire_fighters.tscn

```diff
@@ -38,6 +40,12 @@ tile_size = Vector2i(32, 32)
 physics_layer_0/collision_layer = 2
 sources/0 = SubResource("TileSetAtlasSource_n1sel")
 
+[sub_resource type="Resource" id="Resource_3xfq4"]
+script = ExtResource("14_ymaye")
+function_name = "exit_minigame"
+hotkey = "escape"
```

Comment:
Pressing `escape` on the fire fighters minigame will exit the minigame with this debugging shortcut tool.